docs/getting-started/installing-service-binding.md

```diff
@@ -6,11 +6,182 @@ sidebar_position: 1
+  creationTimestamp: "2021-05-17T07:09:48Z"
+  generation: 1
+  managedFields:
+  - apiVersion: operators.coreos.com/v1alpha1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:annotations:
+          .: {}
+          f:kubectl.kubernetes.io/last-applied-configuration: {}
+      f:spec:
+        .: {}
+        f:services: {}
+    manager: kubectl
+    operation: Update
+    time: "2021-05-17T07:09:48Z"
+  - apiVersion: operators.coreos.com/v1alpha1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:spec:
+        f:detectBindingResources: {}
+      f:status:
+        .: {}
+        f:conditions: {}
+        f:secret: {}
+    manager: service-binding-operator
+    operation: Update
+    time: "2021-05-17T07:09:48Z"
+  name: binding-request-backend
+  namespace: default
+  resourceVersion: "5795"
+  uid: 3c2114c9-311d-43a9-92e1-640179ab673a
+spec:
+  services:
+  - group: stable.example.com
+    kind: Backend
+    name: backend-demo
+    version: v1
+status:
+  conditions:
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    message: the server could not find the requested resource
+    reason: ServiceNotFound
+    status: "False"
+    type: CollectionReady
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "False"
+    type: InjectionReady
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "False"
+    type: Ready
+  secret: ""
+
+```
+
+## Install on Openshift
+
+1. Login to your Openshift Console as a Cluster Admin to install the Service Binding Operator.
+
+![Developer Console](../../static/img/docs/console-login.png)
+
+2. Navigate to the `Operators` -> `OperatorHub` in the OpenShift console.
+
+2. Select `Developer Tools` category.
+
+![Service Binding Operator](../../static/img/docs/operatorhub.png)
+
+3. Select the `Service Binding Operator`:
+
+![Service Binding Operator](../../static/img/docs/operator-hub-sbo-screenshot.png)
+
+4. Select `beta` channel.
+
+5. Select `All namespaces on the cluster` installation mode.
+
+6. Select `Automatic` approval strategy.
+
+![Service Binding Operator](../../static/img/docs/operator-installation.png)
+
+7. Click on `Install`.
+
+![Service Binding Operator](../../static/img/docs/sb_successful_install.png)
+
+8. To verify the installation, click on the `YAML` option in the `Topology` tab in the Developer perspective.
+
+![Service Binding Operator](../../static/img/docs/sb_add_yaml.png)
+
+9. Add the following YAML in the editor:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+        name: binding-request
+spec:
+        services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
```

Comment:
IMHO, this does not really validate installation - it just show that `ServiceBinding` resource can be created, but nothing else. BTW, the pasted yaml refer the wrong `apiVersion` - not used for some time already.